### ISSUE TITLE
COVID-19 Cases Worldwide: add a history

### ISSUE THREAD

**Issue by muescha** [maxMaxineChen/COVID19-Worldwide-Stats#6]
add a small line with history (which can be stored in cookies)

as a line below the graph

`last: US, RU, DE, BB, GE`

tooltip can show the full country names

motivation:
when i look through some countries they are on different pages, when i like to see them often then i have to go always to the deep page again to see the values

**Comment by maxMaxineChen** [comment]
Good idea. Either record history or add a button to record a country as a favorite is good for me.
